projects/lights_control/inc/lights_can.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "can.h"
+#include "lights_events.h"
+
+#define CAN_NUM_RX_HANDLERS 1
+
+typedef enum {
+  LIGHTS_ACTION_SIGNAL_RIGHT = 0,
+  LIGHTS_ACTION_SIGNAL_LEFT,
+  LIGHTS_ACTION_SIGNAL_HAZARD,
+  LIGHTS_ACTION_HORN,
+  LIGHTS_ACTION_HEADLIGHTS,
+  LIGHTS_ACTION_BRAKES,
+  LIGHTS_ACTION_STROBE,
+  LIGHTS_ACTION_SYNC,
+  NUM_ACTION_ID
+} LightsActionID;
+
+// initializes the lights_can module.
+// needs to know whether it's the front light board or
```

Comment:
Nit: Comment width should be 100 weird line breaks make it hard to read.

Reply:
I've added this to our coding standards for future reference.